tests/unit/util/utils_test.cpp

```diff
@@ -1,8 +1,23 @@
 
-#include "gtest/gtest.h"
-
 #include "../../../src/util/Utils.h"
 
+#include <vector>
```

Comment:
Lets add standard license header to this file.

Reply:
Done

Reply:
Ok